shared-scripts.sh

```diff
@@ -319,7 +319,6 @@ function runE2eTest {
 
 function _deploy {
   echo "Deploy"
-  loadCacheFromBuildJob
```

Comment:
Is this removed because there is an additional `yarn production-build` running in deploy?

Reply:
Yes we probably don't need it. Tag release atleast works without it.